components/smee/production/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
(optional) I assume you are provisioning for production eventually but currently this file is not used, you could remove the production folder.